src/components/Reports/ReportsDailyVisits/ReportsDailyVisits.js

```diff
@@ -18,23 +18,54 @@ const ReportsDailyVisits = ({ domainName, dateFrom, dependencies: { datahubClien
       show: false,
     },
     tooltip: {
-      format: {
-        title: (date) => {
-          return intl.formatDate(date, {
+      contents: function(data) {
+        if (data.length) {
+          const date = intl.formatDate(data[0].x, {
             timeZone: 'UTC',
             day: 'numeric',
             month: 'long',
             year: 'numeric',
             weekday: 'long',
           });
-        },
-        value: (value) => {
-          return (
-            intl.formatMessage({
-              id: 'reports_daily_visits.tooltip_page_views',
-            }) + value
-          );
-        },
+
+          const tooltipData = data.reduce((accumulator, item) => {
+            accumulator[item.id] = item.value;
+            return accumulator;
+          }, {});
+
+          return `
+          <div class="c3-tooltip">
+            <div class='tooltip-title'>${date}</div>
+            <div class='tooltip-value'>
+                ${
+                  !tooltipData.withEmail
+                    ? intl.formatMessage({
+                        id: 'reports_daily_visits.tooltip_page_views',
+                      }) + tooltipData.quantity
+                    : ''
+                }
+            </div>
+            <div class='tooltip-value'>
+                ${
+                  !!tooltipData.withEmail
+                    ? intl.formatMessage({
+                        id: 'reports_daily_visits.tooltip_with_email',
+                      }) + tooltipData.withEmail
```

Comment:
What do you think about adding the space here in place of the language file?
```suggestion
                      }) + ' ' + tooltipData.withEmail
```

I Feel that the spaces are error-prone: if change the layout we should also change the language file, that is not fine.

![image](https://user-images.githubusercontent.com/1157864/65160725-a6897600-da0c-11e9-9b3e-a61011afac02.png)

Reply:
I'm not so sure about this, take a look at the line above, we used the same criteria: leave a blank space in the text. I think we should be consistent in the way we handle this issues.

Reply:
Ok, it is just that I would prefer being consistent in a different way, I think that if something is a label should not have space after and if it is a sentence it should not be concatenated. 

I think that it would be a better model of our intention. But, it is not necessary go deeper by the moment, it is not so important.

Thanks for the patience.